modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -7,13 +7,11 @@
 [id="security-context-constraints-psa-synchronization_{context}"]
 = Security context constraint synchronization with pod security standards
 
-{product-title} includes link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Globally, the `privileged` profile is enforced, and the `restricted` profile is used for warnings and audits.
-
 In addition to the global pod security admission control configuration, a controller exists that applies pod security admission control `warn` and `audit` labels to namespaces according to the SCC permissions of the service accounts that are in a given namespace.
```

Comment:
```suggestion
In addition to the global pod security admission control configuration, you can use a controller to apply the pod security admission control `warn` and `audit` labels to namespaces according to the SCC permissions of the service accounts that are in a given namespace.
```
Just a suggestion - this might be a little more engaging if you address the reader directly.

Reply:
So the controller isn't something that the user directly uses, it just happens. But yeah I wasn't a fan of the "exists" part. I will look to reword

Reply:
Ah. Thanks for the clarification. Maybe "a controller applies pod security admission control..."?

Reply:
Yep I agree that should work thanks!